schema/PublicationIssue.md

```diff
@@ -0,0 +1,66 @@
+| pageEnd               | `string` \| `integer`                                                           | The page on which the work ends; for example "138" or "xvi".                                                            | [PublicationIssue](./PublicationIssue.html) |
+| pageStart             | `string` \| `integer`                                                           | The page on which the work starts; for example "135" or "xiii".                                                         | [PublicationIssue](./PublicationIssue.html) |
+| pagination            | `string`                                                                        | Any description of pages that is not separated into pageStart and pageEnd; for example, "1-6, 9, 55" or "10-12, 46-49". | [PublicationIssue](./PublicationIssue.html) |
+| parts                 | `array<`​[`CreativeWork`](./CreativeWork.html)​`>`                              | Elements of the collection which can be a variety of different elements, such as Articles, Datatables, Tables and more. | [CreativeWork](./CreativeWork.html)         |
+| publisher             | [`Person`](./Person.html) \| [`Organization`](./Organization.html)              | A publisher of the CreativeWork.                                                                                        | [CreativeWork](./CreativeWork.html)         |
+| text                  | `string`                                                                        | The textual content of this creative work.                                                                              | [CreativeWork](./CreativeWork.html)         |
+| title                 | `string`                                                                        | The title of the creative work.                                                                                         | [CreativeWork](./CreativeWork.html)         |
+| url                   | `string<uri>`                                                                   | The URL of the item.                                                                                                    | [Thing](./Thing.html)                       |
+| version               | `string` \| `number`                                                            | The version of the creative work.                                                                                       | [CreativeWork](./CreativeWork.html)         |
+
+:::
+
+A part of a successively published publication such as a periodical or publication volume, often numbered, usually
+containing a grouping of works such as articles.
+
+# Examples
+
+An example of an issue of the journal Nature is below. Note that the issue has an `issueNumber` and `isPartOf` a volume that, in turn, `isPartOf` the [`Periodical`](./Periodical.html) with the title "Nature". 
```

Comment:
I think it is work providing a little explanation for each of our examples. It helps people unfamiliar with the schema, or JSON for that matter, understand the point of each.